addons/io_scene_gltf2/blender/exp/gltf2_blender_extract.py

```diff
@@ -408,10 +409,14 @@ def __get_positions(blender_mesh, key_blocks, armature, blender_object, export_s
 
 def __get_normals(blender_mesh, key_blocks, armature, blender_object, export_settings):
     """Get normal for each loop."""
-    blender_mesh.calc_normals_split()
-
     normals = np.empty(len(blender_mesh.loops) * 3, dtype=np.float32)
```

Comment:
You can push this down into the else block. It doesn't need to be allocated in the if case.